releasenotes/notes/improve-mappers-b55cb0ca5fd656e4.yaml

```diff
@@ -0,0 +1,18 @@
+---
+features:
+  - |
+    The class :class:`.second_q.mappers.ModeBasedMapper` has been added to implement mode based
+    mapping via a Pauli table (previously part of :class:`.second_q.mappers.QubitMapper`).
+upgrade:
+  - |
+    The mode based mappers :class:`.JordanWignerMapper`, :class:`.BravyiKitaevMapper`,
+    :class:`.ParityMapper`, and :class:`.DirectMapper` are now inheriting also from the new
+    class :class:`.ModeBasedMapper` to implement the :meth:`.ModeBasedMapper.mode_based_mapping`
+    method.
+  - |
+    :meth:`.ModeBasedMapper.pauli_table` is now an instance method of all :class:`.ModeBasedMapper`
+    subclasses (previously a class method of :class:`.QubitMapper`)
+  - |
+    Caching has been removed from the parent classes :class:`.QubitMapper` and
+    :class:`.ModeBasedMapper` and is now implemented exclusively for the
+    :meth:`.ModeBasedMapper.pauli_table` methods in the child classes.
```

Comment:
You should also note the methods which were previously part of the public API of `QubitMapper` have been removed from it. This is unlikely to affect anyone (since it will not have worked on mappers that didn't use them) but it should be mentioned.

Reply:
Good point. Done.